boards/shields/mcp2515/adafruit_can_picowbell.overlay

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023 Joseph Yates
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&spi0{
```

Comment:
spi0 is not a well-known connector, see https://docs.zephyrproject.org/latest/hardware/porting/shields.html#board-compatibility. Not sure what it would be for rpi_pico. Is there a common pattern what spi instance is used by the rpi_pico shields?

Reply:
I couldn't find any other "spi"'s or aliases for a nodelabel for spi0 for rpi_pico. I can add one to the "rpi_pico.dts" file and name it "pico_spi". Would also need to edit the "rpi_pico_w.dts" to ensure it works for both boards

Reply:
Is there a common pattern of what SPI is used by the rpi_pico shields? If so, then a nodelabel like pico_spi is the right approach, otherwise I have no idea how to handle it generically, perhaps add nodelabels spi0, spi1 ....

Reply:
I couldn't find any pico shields so I have edited "boards/arm/rpi_pico/rpi_pico-common.dtsi" and added a nodelabel. I have also changed the shield overlay to comply with the node label :)